### PR TITLE
feat: web wallet cookies config

### PR DESCRIPTION
**Pull request by ntheile**
New Config to get cookies working in the web wallet charts

- [x] to be merged after kratos and oathkeeper charts PRs
- [x] to be merged after the backend PR https://github.com/GaloyMoney/galoy/pull/2254/ 
- [x] TODO remove ntheile custom docker images (used just for testing)

### CODE REVIEW

File: dev/README.md [hunk 1]
```diff
@@ -21,10 +21,10 @@ Currently successfully brings up charts - no guarantee that everything is workin
 
 ### Test
 
-#### Forward the galoy API port:
+#### Forward the graphl api ingress port:
```
Comment (krtk6160) [inline dev/README.md:24]:
```suggestion
#### Forward the ingress nginx controller port:
```

File: dev/Makefile [hunk 2]
```diff
@@ -99,6 +99,18 @@ redeploy-signet-monitoring:
 	$(TF) apply -var="bitcoin_network=signet" \
 				-var="name_prefix=galoy-sig" -auto-approve
 
-
 fmt:
 	$(TF) fmt -recursive
+
+plan:
+	$(TF) plan -sensative -var="bitcoin_network=regtest" -var="name_prefix=galoy-dev"
```
Comment (openoms) [inline dev/Makefile:106]: probably a typo, but there is also no -sensitive option for terraform plan.
did you find this command useful at all?

File: dev/Makefile [hunk 3]
```diff
@@ -99,6 +99,18 @@ redeploy-signet-monitoring:
...
+
+review:
+	$(TF) show -json | jq
+
+delete:
+	kubectl delete namespace $(ns) --force
+
+apply:
```
Comment (openoms) [inline dev/Makefile:114]: this is the same as `deploy`

File: dev/Makefile [hunk 4]
```diff
@@ -99,6 +99,18 @@ redeploy-signet-monitoring:
...
 	$(TF) fmt -recursive
+
+plan:
+	$(TF) plan -sensative -var="bitcoin_network=regtest" -var="name_prefix=galoy-dev"
+
+review:
+	$(TF) show -json | jq
+
+delete:
+	kubectl delete namespace $(ns) --force
```
Comment (openoms) [inline dev/Makefile:112]: $(ns) is not defined

File: dev/bitcoin/bitcoind-regtest-values.yml [hunk 5]
```diff
@@ -21,4 +21,4 @@ bitcoindCustomConfig:
   bind: 0.0.0.0
   rpcbind: 0.0.0.0
   rpcallowip: 0.0.0.0/0
-  fallbackfee: 0.0002
+  #fallbackfee: 0.0002
```
Comment (openoms) [inline dev/bitcoin/bitcoind-regtest-values.yml:24]: did this cause any issues?

context from bitcoind --help:
```
  -fallbackfee=<amt>
       A fee rate (in BTC/kvB) that will be used when fee estimation has
       insufficient data. 0 to entirely disable the fallbackfee feature.
       (default: 0.00)
```

File: dev/README.md [hunk 6]
```diff
@@ -35,29 +35,47 @@ Currently successfully brings up charts - no guarantee that everything is workin
...
     ```
-    {"data":{"me":{"phone":"+59981730222"}}}
+    {"data":{"me":{"phone":"+16505554321"}}}
     ```
 #### Test the graphql-admin api
 
 1. get session token
     ```
-    curl -ksS 'https://localhost:8080/admin/graphql' -H 'Content-Type: application/json' -H 'Accept: application/json' --data-binary '{"query":"mutation login($input: UserLoginInput!) { userLogin(input: $input) { authToken } }","variables":{"input":{"phone":"+59981730222","code":"111111"}}}' | jq '.data.userLogin.authToken'
+    curl -ksS 'http://localhost:4002/admin/graphql' -H 'Content-Type: application/json' -H 'Accept: application/json' --data-binary '{"query":"mutation login($input: UserLoginInput!) { userLogin(input: $input) { authToken } }","variables":{"input":{"phone":"+16505554321","code":"321321"}}}' | jq '.data.userLogin.authToken'
```
Comment (openoms) [inline dev/README.md:53]: will need to change the phone number and code in the sample request above as well https://github.com/GaloyMoney/charts/pull/2538/files#diff-e14025c0fa40d4857e4b40fc96ea5ee995afe300014626b68cb55a479fa5b8fcR31

File: dev/README.md [hunk 7]
```diff
@@ -35,29 +35,47 @@ Currently successfully brings up charts - no guarantee that everything is workin
...
     ```
     "Beo2lGQZO4nZKzIyZRlVFIQ6jAeo9bNu"
     ```
 
+#### Test the web wallet
+
+1. port forward the web wallet on 3000
+    ```
+    web_wallet=$(kubectl -n galoy-dev-addons get pods | grep web-wallet-mobile | awk '{print $1}')
+    kubectl -n galoy-dev-addons port-forward --address 0.0.0.0 $web_wallet 3000:3000
```
Comment (openoms) [inline dev/README.md:66]: can just use:
```suggestion
    kubectl -n galoy-dev-addons port-forward --address 0.0.0.0 svc/web-wallet-mobile 3000:80
```
it ia quicker and stays on when the pod's name is changing

File: dev/addons/admin-panel-values.yml [hunk 8]
```diff
@@ -0,0 +1,3 @@
+adminPanel:
+  graphqlUrl: http://api:4002/graphql
```
Comment (openoms) [inline dev/addons/admin-panel-values.yml:2]: Needed to change this to
`http://localhost:4002/graphql`
to be abale to pass the captcha on the admin panel.
(as in the readme forwarded the ports 4002 (http ingress), 3000 (web-wallet) and 3001 (admin panel))

able to log in, but when querying a phone number in the admin panel getting a 400 error:
image

File: dev/Makefile (hunk 4 above)
Comment (ntheile) [reply dev/Makefile:112]: yeah i made a helper because i can never remember the delete command. example usage:

make delete ns=galoy-dev-addons

Comment (openoms) [reply dev/Makefile:112]: I presume this is similar to the redeploy usecases, so could add instead:

```
redeploy-addons:
	-kubectl delete namespace galoy-dev-addons --force
	cd addons
	$(TF) apply -var="bitcoin_network=regtest" \
				-var="name_prefix=galoy-dev" -auto-approve
```

File: dev/addons/admin-panel-values.yml (hunk 8 above)
Comment (ntheile) [reply dev/addons/admin-panel-values.yml]: removed admin panel stuff for a follow up PR

**Review by dolcalmi — APPROVED** [review]

**Pull request merged**